TranslatorReasonerAPI.yaml

```diff
@@ -579,3 +594,72 @@ components:
         URI.
       externalDocs:
         url: https://www.w3.org/TR/2010/NOTE-curie-20101216/
+    MetaKnowledgeGraph:
+      type: object
+      description: >-
+        Knowledge-map representation of this TRAPI web service.
+      properties:
+        nodes:
+          type: object
+          description: >-
+            Collection of node categories provided by this TRAPI web service,
+            indexed by Biolink class CURIEs.
+          additionalProperties:
+            $ref: '#/components/schemas/KMNode'
+        edges:
+          type: array
+          description: >-
+            List of edges/predicates provided by this TRAPI web service.
+          items:
+            $ref: '#/components/schemas/KMEdge'
+          minItems: 1
+    KMNode:
```

Comment:
We should probably rename these to MKNode and MKEdge, or just MNode and MEdge.

Reply:
Or maybe MetaNode, MetaEdge to complement MetaKnowledgeGraph